sbt

```diff
@@ -278,9 +281,36 @@ acquire_sbt_jar () {
   } || {
     sbt_jar="$(jar_file "$sbt_version")"
     download_url "$(make_url "$sbt_version")" "$sbt_jar"
+    [[ -n "${skipVerify}" ]] || verify_sbt_jar "$sbt_jar"
```

Comment:
Let's do things incrementally.  Let's skip gpg key retrieving, verifying and skipping for now and just do the HTTPS change and the MD5 checking.